wagtail/admin/templates/wagtailadmin/shared/shared_avatar.html

```diff
@@ -0,0 +1,12 @@
+{% load wagtailadmin_tags %}
```

Comment:
It would be good to add some basic content here to describe this template (similar to other shared templates)

```html
{% comment "text/markdown" %}
    Displays a user avatar using the avatar template
    Variables this template accepts:

    - `classname` - if present, adds classname to the root class list
    - `user` - A user object to use for the avatar image
    - `size` (string?) - small, large, square
    - `tooltip` (string?) - Modifier classes
{% endcomment %}
```